py/mcts_viz/web_viz.py

```diff
@@ -150,4 +152,4 @@ def update_graph(index, graph_snapshots):
     return figure
 
 if __name__ == '__main__':
-    app.run_server(debug=True)
\ No newline at end of file
+    app.run_server(debug=True, port=8051)
```

Comment:
Can you add this port to `EXPOSED_PORTS` in `run_docker.py`? This will allow us to view the viz when launched on runpod.io